finagle-mysql/src/main/scala/com/twitter/finagle/mysql/CanBeParameter.scala

```diff
@@ -71,6 +71,14 @@ object CanBeParameter {
     }
   }
 
+  implicit val bigIntCanBeParameter = {
+    new CanBeParameter[BigInt] {
+      def sizeOf(param: BigInt) = 8
+      def typeCode(param: BigInt) = Type.LongLong
+      def write(writer: MysqlBufWriter, param: BigInt) = writer.writeBytes(param.toByteArray)
```

Comment:
I think a `throw` is fine in this case, although I don't know where that exception will surface off hand. Also, I think you need to make sure you write *exactly 8 bytes* and right now you will write less in the majority of cases:
```scala
scala> BigInt(10).toByteArray
res1: Array[Byte] = Array(10) // only one byte!
```